checkov/main.py

```diff
@@ -282,6 +284,10 @@ def run(self, banner: str = checkov_banner, tool: str = checkov_tool, source_typ
                     logger.debug('Using API key and not --include-all-checkov-policies - only running platform policies '
                                  '(this is the default behavior, and this message is just for debugging purposes)')
 
+            # set parallelization type for CLI runs to 'spawn' mode,
+            # this can be adjusted by setting the env var CHECKOV_PARALLELIZATION_TYPE
+            parallel_runner.type = ParallelizationType.THREAD if parallel_runner.os == "Windows" else ParallelizationType.SPAWN
```

Comment:
@gruebel Feels like an interesting move here, hopefully it won't backfire with OSs we aren't familiar with

Reply:
We currently only support those 3, but you are still able to override it via the env var anyway 🙂